Makefile

```diff
@@ -247,8 +247,18 @@ test-unit:
 # Container tests have to be run before VM tests according to the current code setup.
 # For more details, see the package-level comment of the e2e package.
```

Comment:
Could you also update this? The order doesn't matter now, and they just need to be run sequentially.